elastic/kibana
# [SIEM] Adds 'Open one signal' Cypress test

## Summary

In this PR we are adding a Cypress test to check that only one signal is opened when more than one are selected.

In order to test it these actions are performed:

1. Open detections page
2. Go to closed signals
3. Checks that the open signal bulk button is disabled
4. Selects more than one closed signal
5. Checks that the open signal bulk button is enabled
6. Opens only the first signal
7. Checks that there is one signal less closed
8. Goes to open signals
9. Checks that there is a open signal

Note that to make the test pass it is expected to have only closed signals. When running the tests in CI mode, this is achieved because it loads using es_arhiver a set of data that only have open signals.
